packages/app/cypress/e2e/cypress-in-cypress.cy.ts

```diff
@@ -104,116 +104,82 @@ describe('Cypress in Cypress', { viewportWidth: 1500, defaultCommandTimeout: 100
       })
     })
 
-    it(`scales the AUT correctly in ${testingType}`, () => {
-      const assertNoScaleShown = () => {
-        // check that no message about scale % is shown,
-        // meaning the AUT is at 100% scale
-        cy.contains('%)').should('not.exist')
-      }
-
-      cy.scaffoldProject('cypress-in-cypress')
-      cy.findBrowsers()
-      openProject(testingType)
-      cy.withCtx((ctx) => {
-        ctx.coreData.localSettings.preferences.reporterWidth = 800
-        ctx.coreData.localSettings.preferences.specListWidth = 250
-        ctx.coreData.localSettings.preferences.isSpecsListOpen = false
-      })
-
-      cy.startAppServer(testingType)
-      cy.visitApp()
-      cy.specsPageIsVisible()
+    describe('dragging panels', () => {
```

Comment:
I split these tests out because...these tests are bonkers. It seems like it was originally testing the AUT scale %, but then had a bunch of UI assertions just tacked onto it. 

Additionally, I removed a test around testing the 200 reporter width at a 500 width screen. The UI is so smooshed, it's a bit ridiculous to even test the UI at that size.